src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -474,6 +474,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_IsPatternAlways:
                 case ErrorCode.WRN_SwitchExpressionNotExhaustiveWithWhen:
                 case ErrorCode.WRN_SwitchExpressionNotExhaustiveForNullWithWhen:
+                case ErrorCode.WRN_RecordNamedDisallowed:
```

Comment:
Shouldn't this warning level be 5 and be introduced as part of the C# 9.0 warning waves?

Reply:
No, this is tied to language version, not warn version.

---
In reply to: [476371816](https://github.com/dotnet/roslyn/pull/47094#discussion_r476371816) [](ancestors = 476371816)